### PR TITLE
Package upgrade

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *Global upgrade of the package.*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : JimboJoe
- [x] **Approval (LGTM)** : JimboJoe
- [x] **Approval (LGTM)** : Kay0u
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/kanboard_ynh%20package_upgrade%20(Official)/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -72,82 +66,90 @@ ynh_clean_setup () {
...
 
-# Move old app dir
-mv "$final_path" "$final_path.old"
-
+# Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source "$final_path"
-mkdir -p "$final_path"/sessions
 
-# restore data
-cp -a "$final_path.old/data" "$final_path"
```
Comment (kay0u) [inline scripts/upgrade]: Why did you delete that part?

Comment (maniackcrudelis) [reply scripts/upgrade]: Duplicating the directory isn't needed. We can overwrite the whole directory directly.
`data` and `plugins` does not contains files in the archive that would delete anything in an instance already used.

Comment (kay0u) [reply scripts/upgrade]: Ok!

**Comment by JimboJoe** [comment]
Will be merged in 3 days.
